promql/functions.go

```diff
@@ -14,12 +14,14 @@
 package promql
 
 import (
+	"fmt"
 	"math"
 	"regexp"
 	"sort"
 	"strconv"
 	"strings"
 	"time"
+	_ "time/tzdata"
```

Comment:
Thank you. I think we should handle this logic in the main packages, and offer a way to pass a timezone zip file as a flag. Indeed, the time definitions in tzdata are outdated. And will always be.

Let's instead offer to load a timezone zip file, document it properly, until there is a better solution for windows users.

Reply:
I would prefer to always embed the tzdata and then allow overriding it with a flag and a pointer to a zip file, or what would speak against embedding it?

Reply:
The issue is that it is out of date. There have been multiple changes since the inclusion in golang. Prometheus is used worldwide, I think we should not provide outdated information to our users.

Reply:
regarding your comment: yes, it would need to go in a pkg in pkg/ and be used in prometheus and promtool.